airflow/utils/dag_processing.py

```diff
@@ -308,6 +369,249 @@ def file_path(self):
         raise NotImplementedError()
 
 
+class DagParsingStat(object):
+    def __init__(self,
+                 file_paths,
+                 all_pids,
+                 done,
+                 all_files_processed,
+                 result_count):
+        self.file_paths = file_paths
+        self.all_pids = all_pids
+        self.done = done
+        self.all_files_processed = all_files_processed
+        self.result_count = result_count
+
+
+class DagParsingSignal(object):
```

Comment:
This is either some Immutable Singleton or
```python
DagParsingSignal = namedtuple('DagParsingSignal',
                             'AGENT_HEARTBEAT MANAGER_DONE TERMINATE_MANAGER END_MANAGER')(
                             'agent_heartbeat', 'manager_done', 'terminate_manager', 'end_manager')```

Reply:
Or Python3 Enum, with https://pypi.org/project/enum34/ as the backport